content/news/XXXX-XX-XX-mixxx-2-3-6-released.md

```diff
@@ -0,0 +1,75 @@
+title: Mixxx 2.3.6 released
+authors: Daniel Schürmann
+tags: 2.3, 2.3.6, release announcement
+status: draft
+comments: yes
+
+Welcome the Mixxx release version 2.3.6, the final maintenance release before releasing the upcoming 2.4.0 with exiting new features.
+
+This small bug fix release may appear insignificant, but it's a testament to the rock-solid quality we bring to every user to ensure a reliable mixing experience.
+
+This release improves, among other things, some details with looping, allows playing tracks exceeding 6 hours and fixes rare crashes in exceptional situations.
+
+You can download the new version from https://mixxx.org/download/ There, you will also find the 2.4&nbsp;beta to test the new features that will be released soon.
```

Comment:
this is not turned into a hyperlink, at least not in the preview
https://deploy-preview-303--mixxx-website.netlify.app/news/2023-08-12-mixxx-2-3-6-released/